### DISCUSSION TITLE
Want stats for them to be generated separately into two different html files

### DISCUSSION THREAD

**Discussion by zhangzhuang15** [webpack-contrib/webpack-bundle-analyzer#635] [Q&A] [answered]
I have two entry point, for example `app1.js` and `app2.js`; I compile them together, and I want stats for them to be generated separately into two different html files. Any solution ? I'm grateful to any response.

**Comment by th0r** [comment, marked as answer]
You can try to use [excludeAssets](https://github.com/webpack-contrib/webpack-bundle-analyzer?tab=readme-ov-file#options-for-plugin) option and analyze your bundle twice, keeping only one entry point in each run.

**Reply by zhangzhuang15** [reply]
Thank you. But I'm just curious about why not provide an option to do that, is there special design wisdom ?

**Reply by th0r** [reply]
Because you was the first one who needs this feature. Also it's not clear to me what to do with lazy chunks in this case.
